docs/_sidebar.md

```diff
@@ -50,6 +50,8 @@
     - [Pub/Sub API](en/building_blocks/pubsub/reference.md)
     - [RPC API](en/building_blocks/rpc/reference.md)    
     - [Configuration API](en/building_blocks/configuration/reference.md)
+    - [Component Ref](en/building_blocks/component_ref/component_ref.md)
```

Comment:
How about moving it to the contribution category? Since this feature is for contributors, not for app developers
![image](https://user-images.githubusercontent.com/26001097/197370572-e49ee1d7-1cd8-4ff8-baee-054fbc41cfd1.png)

Reply:
sure

Reply:
hi, it seems unresolved?

Reply:
sorry, forget to push..